controllers/hetznercluster_controller.go

```diff
@@ -267,9 +241,9 @@ func (r *HetznerClusterReconciler) reconcileNormal(ctx context.Context, clusterS
 	return reconcile.Result{}, nil
 }
 
-// setControlPlaneEndpoint updates hetznerCluster.Spec.ControlPlaneEndpoint and returns whether the hetznerCluster is ready or not.
-func setControlPlaneEndpoint(hetznerCluster *infrav1.HetznerCluster) bool {
+func processControlPlaneEndpoint(hetznerCluster *infrav1.HetznerCluster) {
 	if hetznerCluster.Spec.ControlPlaneLoadBalancer.Enabled {
+		// setControlPlaneEndpoint updates hetznerCluster.Spec.ControlPlaneEndpoint and returns whether the hetznerCluster is ready or not.
```

Comment:
I think this comment got to the wrong line. you can probably remove it

Reply:
done